applications/nrf5340_audio/src/audio/streamctrl.h

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright (c) 2018 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+/** @file streamctrl.h
+ *  @brief Stream control - control transfer and processing of the audio streams
+ *
+ * This is the module responsible for "running the system" after initial setup.
+ * The module assumes that before _start() is called
+ * - hw has been initialized
+ * - the ble subsystem has been initialized
+ *
+ * After being started, the module will handling setup and control of
+ * user interface inputs, audio processing and audio transport.
+ */
+
+#ifndef _STREAMCTRL_H_
+#define _STREAMCTRL_H_
+
+#include <stddef.h>
+#include <zephyr/kernel.h>
+
+/* State machine states for peer/stream */
+enum stream_state {
+	STATE_STREAMING,
+	STATE_PAUSED,
+};
+
+/** @brief Get current streaming state
+ *
+ * @return      stream_state enum value
```

Comment:
stream_state = Stream state

Reply:
This is the variable name, now changed to strm_state as to not conflict with the name of the enum